app/src/main/java/org/mozilla/fenix/home/blocklist/BlocklistHandler.kt

```diff
@@ -51,6 +52,32 @@ class BlocklistHandler(private val settings: Settings) {
             }
         }
 
+    /**
+     * If the state is set to [RecentSyncedTabState.Success], filter the list of recently synced
+     * tabs by the blocklist. If the filtered list of tabs is empty, change the state to
+     * [RecentSyncedTabState.None]
+     *
+     * @param blocklistHandler The blocklist handler used to filter the recent synced tabs.
+     */
+    @JvmName("filterRecentSyncedTabState")
+    internal fun RecentSyncedTabState.filteredByBlocklist(blocklistHandler: BlocklistHandler) =
```

Comment:
I don't think this function needs to be marked as `internal` still. If we did ever modularize Fenix, I would expect this function to be an accessible part of the API.